iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/function_ref.inl

```diff
@@ -108,7 +109,7 @@ inline void function_ref<ReturnType(ArgTypes...)>::swap(function_ref<ReturnType(
 }
 
 template <class ReturnType, class... ArgTypes>
-// AXIVION Next Line AutosarC++19_03-A2.10.4 : Overload for swap(function_ref, function_ref) as in STL
+// AXIVION Next Line AutosarC++19_03-A2.10.4, AutosarC++19_03-A2.10.5 : Overload for swap(function_ref, function_ref) as in STL
```

Comment:
Yeah, this is confusing. I can't think of any way how the method `swap` could have static storage duration here. It might be related to inline variables which were added in C++17. Must be a false positive.